cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -26,19 +26,39 @@
 from cirq_google.experimental import ops as experimental_ops
 
 
-SYC_GATE_FAMILY = cirq.GateFamily(ops.SYC)
-SQRT_ISWAP_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP)
-SQRT_ISWAP_INV_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP_INV)
-CZ_GATE_FAMILY = cirq.GateFamily(cirq.CZ)
-PHASED_XZ_GATE_FAMILY = cirq.GateFamily(cirq.PhasedXZGate)
-VIRTUAL_ZPOW_GATE_FAMILY = cirq.GateFamily(cirq.ZPowGate, tags_to_ignore=[ops.PhysicalZTag()])
-PHYSICAL_ZPOW_GATE_FAMILY = cirq.GateFamily(cirq.ZPowGate, tags_to_accept=[ops.PhysicalZTag()])
-COUPLER_PULSE_GATE_FAMILY = cirq.GateFamily(experimental_ops.CouplerPulse)
-MEASUREMENT_GATE_FAMILY = cirq.GateFamily(cirq.MeasurementGate)
-WAIT_GATE_FAMILY = cirq.GateFamily(cirq.WaitGate)
+# Gate family constants used in various parts of GridDevice logic.
+_SYC_GATE_FAMILY = cirq.GateFamily(ops.SYC)
```

Comment:
It's not clear to me why we have both `_SYC_GATE_FAMILY` and `_SYC_FSIM_GATE_FAMILY`. Can you explain briefly? Also happy to sync offline in a 1:1